app/src/components/LoginRequired.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div class="page-not-found">
+    <i class="material-icons page-not-found-header u--padding-zero" >lock_person</i>
+    <h2 class="u_margin-bottom-small">Login required</h2>
+    <p>The page you are trying to access requires authentication.
+      Would you like to <a>log in</a> or return to the
+      <router-link to=""><a @click="$router.go(-1)">previous page</a></router-link>?
+    </p>
+  </div>
+</template>
```

Comment:
I'm not quite sure how to integrate this change into the Vue router. Maybe we should make this a separate ticket?

Reply:
The router link can stay static since both the 404 page and this returns a page -1 i.e. the previous page, but we can use a slot to control the paragraph so the text can change. That way we can maintain a single component for both.

Reply:
I mean how to route to this page -- Right now I'm using a component for this so I just import the whole thing into the dataset/user page, but for the 404 page it looks like there's a separate route that it uses: 
```
  { path: '/explorer:notFound(.*)', component: NotFound },
  { path: '/mm:notFound(.*)', component: NotFound },
  { path: '/nm:notFound(.*)', component: NotFound },
  { path: '/:notFound(.*)', component: NotFound }
```
and I'm not quite sure how to modify this to also work for authentication

Reply:
Created a new ticket to handle this